src/Symfony/Component/Messenger/Transport/Sender/SendersLocator.php

```diff
@@ -41,20 +42,35 @@ public function __construct(array $sendersMap, ContainerInterface $sendersLocato
      */
     public function getSenders(Envelope $envelope): iterable
     {
+        if (\count($envelope->all(OverrideSendersStamp::class)) > 0) {
```

Comment:
this can be simplified to (reusing my previous proposal):
`if ($senders = $envelope->all(ViaSenderStamp::class)) {